navigator/backend/src/main/scala/com/digitalasset/navigator/UIBackend.scala

```diff
@@ -57,12 +61,27 @@ abstract class UIBackend extends LazyLogging with ApplicationInfoJsonSupport {
     */
   def defaultConfigFile: Path = Paths.get("ui-backend.conf")
 
+  // Uses $DAML_HOME from the SDK Assistant or falls back to its default location:
+  // - on Windows: %APPDATA%/daml
+  // - on Mac and Linux: ~/.daml
+  // The file called `secret` will be looked in the resolved location
+  private[this] val defaultAccessTokenFile: Path =
+    sys.env
+      .get("DAML_HOME")
```

Comment:
I'm wondering whether we should put this logic into the general daml.yaml (or rather daml-project-config) library?

Reply:
`daml-project-config` is about the project-specific configuration, if we make a library of it it should be something different. Since I'll for sure need this piece of code to make Extractor work with authentication I'd suggest we defer it to that contribution. Sounds good?

Reply:
Sounds good. 👍